block/block.py

```diff
@@ -50,6 +51,10 @@ def block_timestamp(self):
         # gets the time and sets timestamp
         return str(date.datetime.now()).encode('utf8')
 
+    def genesis_timestamp(self):
+        # gets the time and sets timestamp
+        self.timestamp = str("2017-10-08 13:18:06.810644").encode('utf8')

```

Comment:
Did we want the time stamp of the Genesis to be hard coded? This could cause problems if we try to use the date in any way. If not, we could remove this method.